dune-project

```diff
@@ -67,6 +67,8 @@ Goblint includes analyses for assertions, overflows, deadlocks, etc and can be e
     (conf-ruby :with-test)
     (benchmark :with-test) ; TODO: make this optional somehow, (optional) on bench executable doesn't work
     conf-gcc ; ensures opam-repository CI installs real gcc from homebrew on MacOS
+    domain-local-await
+    domain-shims
   )
   (depopts
```

Comment:
I think `domainslib` should be under `depopts` since we're doing conditional compilation based on it.

Reply:
Makes sense, see 05a6869

Reply:
The non-locked opam file also needs to be updated. But the locked one should stay as-is, otherwise Goblint won't install on OCaml 4 anymore.

Reply:
57b2e21